lib/aur-fetch

```diff
@@ -129,6 +94,37 @@ fi | while read -r pkg; do
             exit 1
         fi
     fi
+    # Avoid issues with filesystem boundaries. (#274)
+    export GIT_DIR=$pkg/.git GIT_WORK_TREE=$pkg
```

Comment:
Shouldn't there be an additional `[[ -d $pkg/.git ]]` check starting here to avoid subsequent `git` commands failing, if there is some `$pkg` directory that is not a `git` repository? This is unlikely to happen now that we've removed `tar` downloads, but still.

Reply:
If we want to be really safe, we can do:
```
GIT_DIR=$(git rev-parse --resolve-git-dir "$pkg"/.git) || exit
export GIT_DIR GIT_WORK_TREE=$pkg
```
which should bail even if you have an `mkdir -p $pkg/.git` directory that isn't a git repository.

Reply:
Can't say I'm very fond of this approach where we keep bolting on git commands. But I guess it's less bad than what we have now...

Reply:
I am a bit confused, because that was done in response to your comment... 
Checking for the presence of `$pkg/.git` isn't enough to avoid subsequent git commands failing.
In most cases, It needs at least `.git/refs/`, `.git/objects/` and a valid `HEAD` file, for commands to consider they're are dealing with a real repository.

I am not sure how we could do what we are doing without bolting these git commands.... :confused: